source/includes/README.md

```diff
@@ -0,0 +1,14 @@
+# Booster Documentation
+
+1. [Getting Started](_01-getting-started.md): Setup your environment to start creating Booster applications.
+2. [Architecture](_02architecture.md): Understand the high-level Booster Cloud Framework architecture.
+3. [Commands](_03commands.md): Design the Write API of your application.
+4. [Events](_04events.md): Understand the source of truth of your application.
+5. [Entities](_05entities.md): Design the domain model of your application.
+6. [Read Models](_06read-models.md): Design the Read API of your application.
+7. [Authentication and Authorization](_07auth.md): Limiting access.
```

Comment:
```suggestion
7. [Authentication and Authorization](_07-auth.md): Limiting access.
```

Reply:
This file is outdated and not necessary. I will remove it!

Sorry @tainguyenbui!